tests/integration/api/test_get_edition.py

```diff
@@ -1,10 +1,20 @@
 import pytest
 import requests
+import json
 from .constants import API_URL
 from .utils import assert_response_status
 
+@pytest.fixture(scope="module", autouse=True)
+def seed_test_data():
+    process = SeedTestDataProcess()
+    process.runProcess()
+
+with open('test_data_ids.json', 'r') as f:
+    test_data_ids = json.load(f)
+    SEEDED_EDITION_ID = test_data_ids['edition_id']
+
 @pytest.mark.parametrize("endpoint, expected_status", [
-    ("/editions/1982731", 200),
+    ("/editions/{SEEDED_EDITION_ID}", 200),
```

Comment:
Is there a way to grab the edition from a fixture after it's been seeded and use it here in the parametrize annotation?